build.sbt

```diff
@@ -12,10 +12,10 @@ lazy val jsmessages = project
   .settings(commonSettings: _*)
   .settings(
     name := "play-jsmessages",
-    crossScalaVersions := Seq("2.12.11", "2.13.1"),
+    crossScalaVersions := Seq("2.13.12", "3.3.1"),
```

Comment:
Is it possible to still support Scala 2.12?

Reply:
it looks like Play 2.9 has dropped support for Scala 2.12 https://www.playframework.com/documentation/3.0.x/Highlights29

Reply:
OK!